MetaMask/metamask-extension
# patch: Hide "Add Snap accounts" toggle from experimental settings

## **Description**
Undoes this pr: https://github.com/MetaMask/metamask-extension/pull/21221

Due to Unforeseen issues with our snap partners we will need to remove the "Add Snap accounts" from the main Metamask build. The previous implementation used a time based toggle to enable this feature. To undo this change I have remove the timestamp variable and simply set the toggle to null in main and rendered it in flask. 

## **Related issues**

Fixes: #

## **Manual testing steps**

#### Testing main
1. pull this branch `patch/hide-keyring-snaps-toggle`
2. run `yarn install`
3. run `yarn start`
4. load the `dist/chrome` or `dist/firefox` into your browser via `manage extensions/load unpacked`
5. create a wallet
6. open settings by clicking the 3 dots in the top right corners
7. navigate to `experimental`
8. scroll to the bottom

**EXPECTED**: You _should not_ see `Add Snap accounts`

#### Testing flask
1. pull this branch `patch/hide-keyring-snaps-toggle`
2. 2. run `yarn install`
3. run `yarn start --build-type flask`
4. load the `dist/chrome` or `dist/firefox` into your browser via `manage extensions/load unpacked`, If you previously manually installed MetaMask you can skip this step.
5. create a wallet. If you previously manually installed MetaMask you can skip this step.
6. open settings by clicking the 3 dots in the top right corners
7. navigate to `experimental`
8. scroll to the bottom

**EXPECTED**: You _should_ see `Add Snap accounts`

## **Screenshots/Recordings**

##### MetaMask **Main** build
<img width="424" alt="Screenshot 2023-11-01 at 2 32 37 PM" src="https://github.com/MetaMask/metamask-extension/assets/22918444/f47b3ed6-c9ab-4004-a164-d2e0489b1412">


##### MetaMask **Flask** Build
<img width="511" alt="Screenshot 2023-11-01 at 1 51 52 PM" src="https://github.com/MetaMask/metamask-extension/assets/22918444/33b5fe9e-95d1-4f84-8a21-262aa64bf89a">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
